app/views/users/index.html.erb

```diff
@@ -1,6 +1,13 @@
-<h1>A list of Users</h1>
-<p>Sample user</p>
-<p>Sample user</p>
-<p>Sample user</p>
-<p>Sample user</p>
-<p>Sample user</p>
\ No newline at end of file
+<div class="users p-5 align-items-center">
+  <% @users.each do |user| %>
+    <%= link_to user do %>
+      <div class="user rounded row align-items-center m-5 p-2">
+        <%= image_tag(user.photo, alt: "user image", class: "col-3 user-images") %>
+        <div class="justify-content-start mb-5 col-6">
+            <h1><%= user.name %></h1>
+        </div>
+        <h6 class="align-items-end col-3">Number of posts: <%= user.posts_counter %></h6>
+      </div>
+    <% end %>
```

Comment:
-  [ ] **[OPTIONAL]** To keep our views clean and more organized I recommend you move this user details into a partial.